src/services/io/podio/CMakeLists.txt

```diff
@@ -8,18 +8,14 @@ print_header(">>>>   P L U G I N :   ${PLUGIN_NAME}    <<<<")       # Fancy prin
 project(podio_project)
 
 # Find dependencies (JANA and spdlog automatically added already)
-find_package(EDM4HEP REQUIRED)
 find_package(EDM4EIC REQUIRED)
 find_package(podio REQUIRED)
 find_package(ROOT REQUIRED COMPONENTS Core Tree Hist RIO EG)
 find_package(fmt REQUIRED)
```

Comment:
I didn't want to touch it previously because it is the place Nathan is working on right now and it is high probably that one will need to clean it up twice. But since it is started, lets then cleanup this file:

- fmt - is not needed (we use spdlog, that embeds fmt)
- find_package(EDM4EIC REQUIRED) and find_package(podio REQUIRED) are not needed as plugin_add_event_model adds them
- find_package(ROOT... should be replaced with plugin_add_cern_root